.github/workflows/pull-request.yaml

```diff
@@ -25,9 +25,9 @@ jobs:
           java-version: ${{ matrix.jdk-version }}
 
       - name: Install clojure tools-deps
-        uses: DeLaGuardo/setup-clojure@master
+        uses: DeLaGuardo/setup-clojure@12.5
```

Comment:
Why fix this to a version?

Especially when you're setting `cli` to `latest` below...

I think it's clearer to keep `cli` as a version (and 1.11.3.x is current but 1.11.4.x is coming).

For `setup-clojure` I'd be okay with `@12` as a compromise but I would prefer that be kept on `@master` since it's really the `cli` version that's important, yes?

Reply:
Thanks for reviewing!

Delaguardo used to recommend @master, but now shows a specific version in his README.
Master can grab wip not ready for prod usage.
@12 is fine with me, if you prefer that.

I prefer `latest` for `cli`, because I like to always automatically test against the latest cli.
And then I show what that actual version is later via a Tools Versions step.

Reply:
But @seancorfield, these are just my preferences. 
I'm just sharing my rationale, if you still feel differently, that's fine, can change.

Reply:
I raised it because elsewhere you mentioned predictable repeatability for CI and this seems to fly in the face of that.

I guess the specific version of the CLI doesn't matter too much since the actual version of Clojure used for testing etc is specified in `deps.edn` -- that does raise a concern tho': right now, there is a limitation in the `clojure -T` machinery that you get whatever version of Clojure the CLI uses -- it doesn't respect the version of Clojure from the `deps.edn` of the tool itself, so that suggests that, at least until that limitation is addressed, it might be worth doing matrix testing against Clojure 1.10.3, 1.11.4, and 1.12.0-rc1? If you think that's worthwhile, create a GH issue.

For this PR, let's go with `@12` and `latest` -- you've partially convinced me 🙂